apps/bluetooth/app.js

```diff
@@ -225,6 +225,16 @@ module.exports = function (activity) {
       var isPlaying = a2dp.getAudioState() === protocol.AUDIO_STATE.PLAYING
       logger.debug(`Start play bt music, mode=${mode}, radio=${a2dp.getRadioState()}`)
       if (mode === protocol.A2DP_MODE.SINK && isConnected && !isPlaying) {
+        setTimer(() => {
```

Comment:
Should the timer be exclusive? i.e. there should be no another timer be set while first timer was not yet committed.

Reply:
function setTimer (callback, timeout) {
    if (timer != null) {
      clearTimeout(timer)
    }
    timer = setTimeout(callback, timeout)
  }